.gitignore

```diff
@@ -16,6 +16,7 @@ zig-out/
 /build/
 /build-*/
 /docgen_tmp/
+.DS_Store
```

Comment:
If you can't be bothered to read the comment above, why should I trust your other work in this PR?

Reply:
I'm sorry I overlooked the description about `gitignore`, I have rolled back the code.